crates/fuel-core/src/state/rocks_db.rs

```diff
@@ -1157,8 +1244,11 @@ mod tests {
             tmp_dir.path(),
             secondary_temp.path(),
             old_columns.clone(),
-            None,
-            512,
+            DatabaseConfig {
```

Comment:
Maybe we can replace all distance with `DatabaseConfig::config_for_tests()`

Reply:
Done